### PR TITLE
Added Vimeo and Youtube albums for rhel, software collections, atomic…

### PR DESCRIPTION
**Pull request by Dantheman720**
…, developer toolset, openshift and satellite.

### CODE REVIEW

**Comment by Dantheman720** [comment]
Learn pages to follow in a separate pull request.

File: products/mobileplatform/_common/product.yml [hunk 1]
```diff
@@ -2,8 +2,6 @@ name: Red Hat Mobile Application Platform
 abbreviated_name: Mobile Application Platform
 current_version: 1.0.0.GA
 intro_youtube: https://www.youtube.com/v=Dw-Z-R2C51U
-youtube_album: https://www.youtube.com/playlist?list=PLf3vm0UK6HKroZy6oqmmS1uBXRbq8QiS9
-vimeo_album: https://vimeo.com/album/3550548
```
Comment (paulrobinson) [inline products/mobileplatform/_common/product.yml]: Why are you removing these playlist?

Comment (Dantheman720) [inline products/mobileplatform/_common/product.yml]: That was a mistake.  Fixing now.

**Comment by Dantheman720** [comment]
retest this please

**Pull request merged**